testbed/stabilitytests/metric_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"testing"
+
+	"go.opentelemetry.io/collector/testbed/testbed"
+	scenarios "go.opentelemetry.io/collector/testbed/tests"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/datareceivers"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/datasenders"
+)
+
+func TestStabilityMetricsOpenCensus(t *testing.T) {
+	scenarios.Scenario10kItemsPerSecond(
```

Comment:
I was envisioning stability test different from Scenario10kItemsPerSecond.
Instead of checking for maximum CPU and RAM limits it would initially run for a few seconds and then would take the average peak value of CPU and peak value of RAM observed and then continue running for longer and see if CPU or RAM will go over the initially observed peak by more than a configured small amount. (Or perhaps we do this only for RAM and for CPU we keep the original monitoring logic).
What you have is fine for initial version but I would want to see a more sophisticated test in the future.

Reply:
Yes, that's much better approach. I was watching  the resource usage during the test. For most of the cases with current setup RAM usage doesn't seem to stabilize at one value. We probably need to introduce a memory limiter and make sure no drops happened, right?
You can see how it went btw https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector-contrib/1279/workflows/4099caea-84f4-4cdd-9d75-2d1e4ed31165/jobs/3528

Reply:
I am not sure why memory limiter is going to make a difference. Is it because it forces a GC when the limit is hit?

Reply:
Yes, I was thinking it may help. I've checked, and it doesn't matter. I'll need to investigate why do we have slowing growing RAM usage. Then I'll change the tests to work the way as you suggested